custom/nutrition_project/ucr/data_sources/supplementary_nutrition_thr_v1.json

```diff
@@ -0,0 +1,200 @@
+{
+  "domains": [
+    "india-nutrition-project"
+  ],
+  "server_environment": [
+    "india"
+  ],
+  "config": {
+    "table_id": "static-suppl_nutrition_thr_v1",
+    "display_name": "Supplementary Nutrition THR V1 (Static)",
+    "referenced_doc_type": "XFormInstance",
+    "configured_filter": {
+      "type": "and",
+      "filters": [
+        {
+          "type": "or",
+          "filters": [
+            {
+              "type": "boolean_expression",
+              "operator": "eq",
+              "expression": {
+                "type": "property_name",
+                "property_name": "xmlns"
+              },
+              "property_value": "http://openrosa.org/formdesigner/4FAB81D4-BBAF-424E-8DB1-96478791DB01"
+            },
+            {
+              "type": "boolean_expression",
+              "operator": "eq",
+              "expression": {
+                "type": "property_name",
+                "property_name": "xmlns"
+              },
+              "property_value": "http://openrosa.org/formdesigner/116A78AD-E839-4BD4-BA13-A534A4A85B3B"
+            }
+          ]
+        },
+        {
+          "type": "boolean_expression",
+          "operator": "eq",
+          "expression": {
+            "type": "property_name",
+            "property_name": "app_id"
+          },
+          "property_value": "b3c758f7c79e47e3a503115894f18503"
```

Comment:
Not blocking. Curious if xmlns filter is applied above why is app filter is added here? Do you expect forms in different apps to have same xmlns? Because people are having multiple copies of app?

Reply:
That is how HQ makes a data source by default when you select a form from an app to make a data source. So i kept the same pattern. I think XMLNS is unique globally so this check does seem redundant but i would confirm that.
Eventually we would need to remove this app id check when we start making linked apps.